devices/pine64-pinephone-braveheart/default.nix

```diff
@@ -13,10 +13,11 @@
     screen = {
       width = 720; height = 1440;
     };
+    rev = if "" == builtins.getEnv "MOBILE_NIXOS_DEVICE_HW_REV" then "1.0" else builtins.getEnv "MOBILE_NIXOS_DEVICE_HW_REV";
```

Comment:
While there is an environment variable that can affect the build for the device name, it should probably be deprecated as this causes build reproducibility issues. This should be done entirely through configuration *if wanted*.